src/OpenTelemetry.Exporter.Console/ConsoleExporter.cs

```diff
@@ -10,7 +10,7 @@ public abstract class ConsoleExporter<T> : BaseExporter<T>
 
     protected ConsoleExporter(ConsoleExporterOptions options)
     {
-        this.options = options ?? new ConsoleExporterOptions();
+        this.options = options;
```

Comment:
Why can't someone pass null here?

Reply:
well what is the intent.

if ConsoleExporterOptions cant be null it should guard and throw on null.

if ConsoleExporterOptions can be null it should be `protected ConsoleExporter(ConsoleExporterOptions? options)`